GitUI/LeftPanel/RepoObjectsTree.cs

```diff
@@ -61,7 +61,6 @@ public RepoObjectsTree()
             mnubtnPullFromRemoteBranch.AdaptImageLightness();
             InitializeComplete();
 
-            ReloadHotkeys();
             HotkeysEnabled = true;
```

Comment:
This flag controls whether hotkeys are loaded or not. I don't know this area enough to appreciate in what side-effects this move would result. This would require an investigation which is not relevant to this change, and would more relevant to the conversion of `HotkeySettingsManager` which I mentioned in https://github.com/gitextensions/gitextensions/pull/11242#discussion_r1349417428.